### PR TITLE
bugfix/17472-searchPointByAngle-polar

### LINKED ISSUE #17472: Spiderweb Shared Tooltip not rendering
#### Expected behaviour
Shared Tooltip should render multiple datapoints for the respective variable.

#### Actual behaviour
Tooltip does not render at all and the following error is thrown in the console.

highcharts_spiderweb

#### Live demo with steps to reproduce

Shared tool tip is no longer working on the demo provided via the Highcharts website. Just hover over a datapoint and you will see the tool tip does not render.
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/polar-spider

#### Product version

Latest version from https://code.highcharts.com

#### Affected browser(s)
Chrome, Firefox, IE.

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #17472, shared tooltip did not work in polar charts.

- [x] test

### CODE REVIEW

File: ts/Series/PolarComposition.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ export declare class PolarSeriesComposition extends Series {
     animate(init?: boolean): void;
     searchPoint: (
         PolarSeriesComposition['kdByAngle'] extends true ?
-            PolarAdditions['searchPointByAngle'] :
+            'searchPointByAngle' :
```
Comment (bre1470) [inline ts/Series/PolarComposition.ts:149]: `'searchPointByAngle'` is now a string literal type. Try `typeof searchPointByAngle`

**Review by bre1470 — CHANGES_REQUESTED** [review]
One thing about typing needs be fixed.

Otherwise it looks good. The only drawback might be that is not accessible anymore, but it does not look like it is used outside of polar anyway.

**Review by bre1470 — APPROVED** [review]
Thanks! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Comment by GabrieleT0** [comment]
it seems that the problem is not solved, in this example it is still present: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/polar-spider

**Comment by bre1470** [comment]
Thank you for the feedback. The changes are not yet released, see https://jsfiddle.net/bre1470/dk6ypz3w/ for the working example based on the current GitHub master.

**Comment by GabrieleT0** [comment]
Thank you so much!

**Comment by GabrieleT0** [comment]
There is a date for the release? I'm doing my bechelor's thesis and I'm using Hightchart, it's very important for me. Thank you.

**Comment by bre1470** [comment]
We expect the next release to be ready this month.

**Pull request merged**